core/services/feeds/service.go

```diff
@@ -1000,35 +1061,13 @@ func (s *service) getAddressAndEVMChainIDFromJob(j *job.Job) (ethkey.EIP55Addres
 	case job.OffchainReporting:
 		address = j.OCROracleSpec.ContractAddress
 		evmChainID = j.OCROracleSpec.EVMChainID
-	case job.OffchainReporting2:
-		eipAddress, addrErr := ethkey.NewEIP55Address(j.OCR2OracleSpec.ContractID)
```

Comment:
This is never used because contractAddress as an `EIP55Address` as well as evmChainID does not exist on `ocr2_oracles_spec`